cmd/wallet-web/src/components/Vaults/AddVaultModal.vue

```diff
@@ -0,0 +1,163 @@
+<!--
```

Comment:
Instead of deleting `cmd/wallet-web/src/components/Modal/AddVault.vue`, can you move it to the new location and rename, please? Otherwise git won't be able to track older changes.

Reply:
```bash
git mv cmd/wallet-web/src/components/Modal/AddVault.vue cmd/wallet-web/src/components/Vaults/AddVaultModal.vue
```